scripts/roundtrip.py

```diff
@@ -41,7 +41,7 @@ def compile(clang, input, output, timeout, options=None):
     cmd.extend([input, "-o", output])
     p = run_cmd(cmd, timeout)
     assert p.returncode == 0, \
```

Comment:
Only use `assert` for things that you optionally want to check, just in case this is run with "optimizations" enabled. I.e. turn these `assert`s into actual `if` conditions that raise exceptions.

Reply:
Changed `assert` to `unittest.TestCase.assertEqual()`. This should hopefully survive optimized runs.